app/resources/assets/sass/_location_report.scss

```diff
@@ -40,14 +40,11 @@
 		}
 		
 		.outer-questions-box {
-			background-image: url('../images/arrow.png');
-			background-repeat: no-repeat;
-			background-position: 0 center;
 			position: absolute;
-			top: 50%;
-			left: 50%;
+			top: 0;
+			right: 0;
+			left:75%;
```

Comment:
I think right: 0; would work better than left: 75%.

For the other problem I mentioned with the box shifting around, you may need to look for different styles applied for different viewport dimensions.